Resources/Prototypes/Entities/Structures/Machines/base_structuremachines.yml

```
@@ -72,6 +76,10 @@
     - machine_board
   - type: LightningTarget
     priority: 1
+  - type: Climbable # mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)